### PR TITLE
docs/dasharo-menu-docs/overview.md: hotkeys section added

### PR DESCRIPTION
**Pull request by filipleple**

### CODE REVIEW

File: docs/dasharo-menu-docs/overview.md [hunk 1]
```diff
@@ -58,6 +58,37 @@ board variant.
   used for setup application. Lists all bootable options and allows one to
   override the boot path.
 
+### Hotkeys
+
+In the standard EDK2 BIOS, the functionality of the F9 and F10 hotkeys is
```
Comment (macpijan) [inline docs/dasharo-menu-docs/overview.md:63]: I do not think we need to compare with that, regular end user may not know what that means. In this documentation we simple describe what is the expected behavior in Dasharo firmware menus.

This paragraph should be rephrased, or maybe even removed if no idea how to rephrase it.

File: docs/dasharo-menu-docs/overview.md [hunk 2]
```diff
@@ -58,6 +58,37 @@ board variant.
...
+limited to certain menus and applies only to the settings within the current
+menu or entry. In Dasharo, these hotkeys are present and effective globally.
+
+#### F9: Reset to Defaults
+
+* The `F9` hotkey is a **global reset option**.
+* Pressing `F9` will reset all BIOS settings across all menus to their default
+values.
+* This action is immediate and applies to all configurable options within the
```
Comment (macpijan) [inline docs/dasharo-menu-docs/overview.md:72]: We have one exception, the secure boot keys will not be reset to defaults.

**Review by macpijan — COMMENTED** [review]
Overall it is what we need, but with some comments

File: docs/dasharo-menu-docs/overview.md (hunk 2 above)
Comment (filipleple) [reply docs/dasharo-menu-docs/overview.md:68]: Change applied [here](https://github.com/Dasharo/docs/pull/709/files#diff-256c66f72451358d31ff2d64488936acff63f53b0b36cbb8e1f0be9b9d5406d0R67)

File: docs/dasharo-menu-docs/overview.md (hunk 1 above)
Comment (filipleple) [reply docs/dasharo-menu-docs/overview.md]: Paragraph [removed](https://github.com/Dasharo/docs/pull/709/files#diff-256c66f72451358d31ff2d64488936acff63f53b0b36cbb8e1f0be9b9d5406d0R61)

**Review by macpijan — APPROVED** [review]

**Pull request merged**